rp_transferto/tasks.py

```diff
@@ -64,6 +67,19 @@ def update_values(org, user_uuid, values_to_update, transferto_response):
     rapidpro_client.update_contact(user_uuid, fields=fields)
 
 
+@app.task(
```

Comment:
I'm not sure if this task is really related to transferto, it should probably move to sidekick.

Reply:
Yeah I didn't want to have the same function repeated in two places. Should I move the task to sidekick but keep the function here, and import it in sidekick?

Reply:
Can we move the function to sidekick/utils and the task to sidekick?